configs/platforms/solaris-10-i386.rb

```diff
@@ -38,6 +38,8 @@
   pkgadd -n -a /var/tmp/vanagon-noask -G -d http://get.opencsw.org/now all;
   /opt/csw/bin/pkgutil -y -i curl rsync gmake pkgconfig ggrep;
   ln -sf /opt/csw/bin/rsync /usr/bin/rsync;
+  # RE-6121 openssl 1.0.2e requires functionality not in sytem grep
+  ln -sf /opt/csw/bin/ggrep /usr/bin/grep;
```

Comment:
What about aix? How is the grep there? Or does that work because /opt/freeware/bin is in PATH?